__tests__/app.test.js

```diff
@@ -14,20 +14,57 @@ afterAll(() => {
 
 describe("GET /api/topics", () => {
   test("should return with a 200 status code", () => {
-    return request(app).get("/api/topics").expect(200)
+    return request(app).get("/api/topics").expect(200);
   });
 
   test("should return with a property of slug and description", () => {
-    return request(app).get("/api/topics").expect(200).then(({body})=>{body.forEach(({
-        description,
-        slug
-    })=>{
-        expect(typeof description).toBe("string")
-        expect(typeof slug).toBe("string")
-    }
-
-    )
-
-    })
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
```

Comment:
In your model send over an object with a key of topics, then you can look in body.topics which makes more sense than just body